src/qt/bitcoingui.cpp

```diff
@@ -853,6 +863,10 @@ bool CreateNewConfigFile(std::string boinc_email)
     myConfig << row;
     row = "addnode=seeds.gridcoin.ifoggz-network.xyz \r\n";
     myConfig << row;
+    row = "addnode=ec2-3-81-39-58.compute-1.amazonaws.com \r\n";
```

Comment:
Line 935 also plugs in some default addnodes at runtime. Do we want to make these match? None of the original nodes in this file seem to be online anymore.

Reply:
Yes. If we can quickly come up with a good list I will standardize. I through our two in there.

Reply:
From the wiki, here are the non-scraper addnodes that responded:

addnode-us-central.cycy.me
gridcoin.ddns.net
tarmoilves.eu